libs/custom-paper-handler/src/flow.test.ts

```diff
@@ -0,0 +1,88 @@
+import { sleep } from '@votingworks/basics';
+import waitForExpect from 'wait-for-expect';
+import makeDebug from 'debug';
+import { ballotFixture } from './ballot_fixture';
+import { getPaperHandlerMachine } from './paper_handler_machine';
+
+const debug = makeDebug('paper-handler:test');
+
+jest.setTimeout(60_000);
```

Comment:
Can we make this not actually take so long?

Reply:
This isn't an actual automated test since it uses a real, unmocked `PaperHandlerMachine` and requires physically inserting the paper. I think it was originally written as a manual smoke test (that's how I've been using it). But I see your point that this slows CI so I'll definitely reduce this and the 10 second `waitForExpect` down below.

Reply:
Reduced timeout and then ignored the test here [cd5448f](https://github.com/votingworks/vxsuite/pull/3472/commits/cd5448f115e1e69eee6e3eb65bbb8eee6ab3cad8)